LICENSE

```diff
@@ -1,28 +1,30 @@
-Copyright (c) Erik Hesselink 2009
+Copyright (c) 2009 Erik Hesselik, 2019 Oleg Grenrus, Ryan Scott
```

Comment:
Still a typo in my last name here, should be 'Hesselink' with an 'n'.

Reply:
Fixed.